spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/properties/PropertiesConfigAdapter.java

```diff
@@ -31,7 +31,7 @@
  */
 public class PropertiesConfigAdapter<T> {
 
-	private T properties;
+	private final T properties;
```

Comment:
Well spotted. This could be applied all the way back to 2.5.x so I've done that in https://github.com/spring-projects/spring-boot/issues/30615.